### DISCUSSION TITLE
Uyuni 2023.10 & OES 23.4

### DISCUSSION THREAD

**Discussion by NMFle** [uyuni-project/uyuni#7886] [General]
Hi,
I posted this in the Gitter chat but I'm not sure where best to post it so I'm doing it here too..

Great that 2023.10 is out, I was waiting for its release to be able to sync the OES 23.4 repos.
The repos are not showing for me when I refresh the product catalogue and whilst investigating I noticed that in
/usr/share/susemanager/scc/additional_products.json the "product_class" of OES 23.4 is OES2-BETA and the 'release_stage" is beta.
I have tried changing these to OES2 and released but OES23.4 still isn't appearing in the list of products.

Are the beta tags causing OES 23.4 to not appear in my product list?  Is there anything else I can do to get it to appear?

Thanks,
Nick

**Comment by mcalmer** [comment]
It is not official blessed by Microfocus. As soon as we get the official OK, we will take care that it works out of the box.
Currently only Microfocus can test this.

**Reply by NMFle** [reply]
Hi.  Many thanks for the reply.  Apologies for the misunderstanding, but what do you mean by blessed?
That Microfocus have to agree to 23.4 being available via Uyuni or something else?  

Thanks,
Nick.

**Reply by mcalmer** [reply]
They are testing this change, and we wait for the OK, before we enable it officially

**Comment by sayingwhat** [comment]
Hello,
This topic is pretty old, sorry about that.
Since 2024.08 and even after migration to 2024.12, the OES repositories are no longer mirrored, or are no longer available as a channel at all.
Is this correct or what should I be aware of?

**Reply by lucidd** [reply]
This is a bug and will be fixed soon by this PR https://github.com/uyuni-project/uyuni/pull/9824
